gradle.properties

```diff
@@ -17,4 +17,4 @@ org.gradle.jvmargs=-Xmx1536m
 # org.gradle.parallel=true
 
 # keep build timestamps in APK
-android.keepTimestampsInApk = true
+android.keepTimestampsInApk = false
```

Comment:
Maybe we can extract apk to verify artifacts instead of adding debug.keystore or droping timestamp.
Just like f-droid verification server does \[[GitLab](https://gitlab.com/fdroid/fdroidserver/blob/master/fdroidserver/common.py#L2780)\].

Reply:
timestamp dropping is a good thing imho, but the debug.keystore is annoying.

So i rewrote the artifacts-verification build and the checksum method to create checksums of the contents of the apk and verify them with a second build.

Reply:
the reason for the timestampsInApk is that I use that option when deploying on f-droid to generate the repo metadata (i.e. to have the correct apk release date). it comes in very handy when recreating the whole repo from scratch.

a reproducible build, of course, may not have this set to ```true```.